tools/setup/emoji/build_emoji

```diff
@@ -245,7 +245,10 @@ def setup_emoji_farms(cache_path: str, emoji_data: List[Dict[str, Any]]) -> None
         shutil.copy2(src_file, dst_file)
 
     def setup_emoji_farm(
-        emojiset: str, emoji_data: List[Dict[str, Any]], alt_name: Optional[str] = None
+        emojiset: str,
+        emoji_data: List[Dict[str, Any]],
+        alt_name: Optional[str] = None,
+        fallback_emoji_data: List[Dict[str, Any]] = list(),
```

Comment:
It looks like you evaded our [semgrep check](https://github.com/zulip/zulip/blob/836db701aa0e70ba0f20fc195f1deaaa49e29bc2/tools/semgrep.yml#L101-L135) for mutable default arguments by changing `[]` to `list()`, but that still has the same [danger](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).

- #23446